website/scripts/send-data-to-vanta.js

```diff
@@ -223,10 +223,16 @@ module.exports = {
               softwareToAdd.name = software.name;
               softwareToAdd.browser = 'FIREFOX';
               softwareToAdd.extensionId = software.name + ' ' + software.version;// Set the extensionId to be the software's name and the software version.
+              if(software.extension_id !== undefined && software.extension_id !== null) {// If the Fleet instance reported an extension_id for the extension, we'll use that value.
+                softwareToAdd.extensionId = software.extension_id;
+              }
```

Comment:
another approach (possibly better?  wouldn't know until bothering to try it, which might or might not make sense) would be to build up key by key.  Like build all `name` first, then the `browser`, and add `if` statements based on source as needed

Reply:
@mikermcneil I updated this script to use that approach